app/models/comment.py

```diff
@@ -10,13 +10,13 @@ class Comment(Base):
     __tablename__ = "comments"
 
     id = Column(Integer, primary_key=True, index=True)
-    content = Column(String, index=True)
+    content = Column(String(2000), nullable=True)
     created_at = Column(DateTime, default=datetime.utcnow)
```

Comment:
instead of datetime.utcnow  values use the sql functions for default values,
In the case of update both create_at and updated_at value change or not